File: Too many Floors.c
#include<stdio.h>
#include<math.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y,a,b,s;
        scanf("%d%d",&x,&y);
        if(x%10==0)
        a=x/10;
        else
        a=(x/10)+1;
        if(y%10==0)
        b=y/10;
        else
        b=(y/10)+1;
        s=abs(a-b);
        printf("%d\n",s);
    }
}

File: Too many Floors.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int a1,b1,c;
            if(a%10==0)
            a1=a/10;
            else
            a1=(a/10)+1;
            if(b%10==0)
            b1=b/10;
            else
            b1=(b/10)+1;
            c=Math.abs(a1-b1);
            System.out.println(c);
        }
    }
}